### ISSUE TITLE
Module members in navigation bar 

### ISSUE THREAD

**Issue by pravdomil** [klazuka/intellij-elm#744]
I'm using navigation bar to switch between files.
And I would be handy to be able to navigate between module members as well.

JavaScript example:
![image](https://user-images.githubusercontent.com/2387356/112874706-8b003080-90c3-11eb-94a6-7409d9b4dc70.png)

**Comment by klazuka** [comment]
Would this work: Press `cmd-F12` on Mac to show the navigation pop-up (https://github.com/klazuka/intellij-elm/blob/master/docs/features/structure-view.md)

**Comment by pravdomil** [comment]
Hi @klazuka, hope you are doing well in new job.
Yes, the the structure view is a alternative.
But I really like the navigation bar (especially using cmd+up shortcut).
And than I can use one interface for both files and members navigation, and that would simplify my workflow.

**Comment by klazuka** [comment]
Makes sense. This would be nice to add. Probably not that hard to do, although I don't know what the plugin hook mechanism looks like.

**Comment by pravdomil** [comment]
Found something here:

https://github.com/search?p=2&q=StructureAwareNavBarModelExtension&type=Code

abstract class StructureAwareNavBarModelExtension:
https://github.com/JetBrains/intellij-community/blob/master/platform/lang-impl/src/com/intellij/ide/navigationToolbar/StructureAwareNavBarModelExtension.kt

kotlin:
https://github.com/JetBrains/kotlin/blob/master/idea/src/org/jetbrains/kotlin/idea/navigationToolbar/AbstractNavBarModelExtensionCompatBase.kt

java:
https://github.com/JetBrains/intellij-community/blob/master/java/java-impl/src/com/intellij/ide/navigationToolbar/JavaNavBarExtension.java
